e2e/cypress/integration/channel_sidebar/channel_sidebar_spec.js

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import users from '../../fixtures/users';
+
+import {testWithConfig} from '../../support/hooks';
+
+import {getRandomInt} from '../../utils';
+
+const sysadmin = users.sysadmin;
+
+describe('Channel sidebar', () => {
+    testWithConfig({
+        ServiceSettings: {
+            ExperimentalChannelSidebarOrganization: 'default_on',
+        },
+    });
+
+    before(() => {
+        cy.apiLogin('user-1');
+
+        cy.visit('/');
+    });
+
+    it('should switch channels when clicking on a channel in the sidebar', () => {
+        // # Start with a new team
+        const teamName = `team-${getRandomInt(999999)}`;
+        cy.createNewTeam(teamName, teamName);
+
+        // * Verify that we've switched to the new team
+        cy.get('#headerTeamName').should('contain', teamName);
+
+        // # Click on Off Topic
+        cy.get('.SidebarChannel:contains(Off-Topic)').should('be.visible').click();
```

Comment:
This failed on me, same case for others using similar selector.  Please check again, thanks.
![Screen Shot 2020-03-17 at 7 48 06 PM](https://user-images.githubusercontent.com/5334504/76853530-6c4c4480-6888-11ea-9e7a-1806fe2cb0fb.png)

Reply:
Are you running with the corresponding `feature/channel-sidebar-organization` branch of the server code? That's needed for the server-side setting to be properly set

Reply:
@saturninoabril @hmhealey As of this afternoon you should be able to use the latest `master` branch, as the new sidebar code has been merged.

Reply:
Thanks, got it.